pom.xml

```diff
@@ -87,6 +87,8 @@
   <properties>
     <!-- Minimum Maven version required by the plugins -->
     <minimalMavenBuildVersion>3.6.3</minimalMavenBuildVersion>
+    <!-- Minimum Java version required by build -->
+    <minimalJavaBuildVersion>8</minimalJavaBuildVersion>
```

Comment:
originally was used `maven.compiler.target` but should be different version for build time and for target